website/src/components/wistia/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+function WistiaVideo({id}) {
+  return (
+
+  	 <React.Fragment>
+		 <div class="wistia_responsive_padding" style="padding:56.25% 0 0 0;position:relative;">
+			<div class="wistia_responsive_wrapper" style="height:100%;left:0;position:absolute;top:0;width:100%;">
```

Comment:
Style tags in React are _slightly_ different than normal html, by accepting objects rather than a string. To resolve, you can change the two style tags from:
`style="padding:56.25% 0 0 0;position:relative;"`
`style="height:100%;left:0;position:absolute;top:0;width:100%;"`
To
`style={{padding:"56.25% 0 0 0",position:"relative"}}`
`style={{height:"100%",left:0,position:"absolute",top:0,width:"100%"}}`

Reply:
Thanks!  Always been confused about React CSS, no longer